### DISCUSSION TITLE
What happened to Vue Components reference in docs?

### DISCUSSION THREAD

**Discussion by miccoh1994** [directus/directus#10226] [Archive]
I think some of the docs were outdated, but still usefull

**Comment by rijkvanzanten** [comment]
We started putting the internal code references in the GitHub Wiki 👍🏻 

https://github.com/directus/directus/wiki

**Reply by MajesteitBart** [reply]
@rijkvanzanten, I thought there were some screenshots too, are those gone now?

**Reply by rijkvanzanten** [reply]
I don't think we ever did screenshots for those system components 🤔

**Reply by rijkvanzanten** [reply]
We did have Storybook at one point (months ago, in v9.0.0-alpha days), but that was dropped a long time ago
